src/mahoji/commands/runecraft.ts

```diff
@@ -122,46 +134,55 @@ export const runecraftCommand: OSBMahojiCommand = {
 
 		let { tripLength } = runeObj;
 		const boosts = [];
-		if (userHasGracefulEquipped(user)) {
-			tripLength -= tripLength * 0.1;
-			boosts.push('10% for Graceful');
+		if (user.isIronman && runners) {
+			runners = false;
+			boosts.push('Ironmen stand on their own');
 		}
 
-		if (user.skillLevel(SkillsEnum.Agility) >= 90) {
-			tripLength *= 0.9;
-			boosts.push('10% for 90+ Agility');
-		} else if (user.skillLevel(SkillsEnum.Agility) >= 60) {
-			tripLength *= 0.95;
-			boosts.push('5% for 60+ Agility');
-		}
+		let inventorySize = 28;
 
-		if (!usestams) {
-			tripLength *= 3;
-			boosts.push('**3x slower** for no Stamina potion(4)s');
-		} else if (user.hasEquippedOrInBank(['Ring of endurance'])) {
-			tripLength *= 0.99;
-			const ringStr = '1% boost for Ring of endurance';
-			boosts.push(ringStr);
-		}
+		if (!runners) {
+			if (!usestams && runeObj.stams) {
+				tripLength *= 3;
+				boosts.push('**3x slower** for no Stamina potion(4)s');
+			} else if (user.hasEquippedOrInBank(['Ring of endurance'])) {
+				tripLength *= 0.99;
+				const ringStr = '1% boost for Ring of endurance';
+				boosts.push(ringStr);
+			}
+			if (userHasGracefulEquipped(user)) {
+				tripLength -= tripLength * 0.1;
+				boosts.push('10% for Graceful');
+			}
 
-		let inventorySize = 28;
+			if (user.skillLevel(SkillsEnum.Agility) >= 90) {
+				tripLength *= 0.9;
+				boosts.push('10% for 90+ Agility');
+			} else if (user.skillLevel(SkillsEnum.Agility) >= 60) {
+				tripLength *= 0.95;
+				boosts.push('5% for 60+ Agility');
+			}
 
-		// For each pouch the user has, increase their inventory size.
-		for (const pouch of Runecraft.pouches) {
-			if (user.skillLevel(SkillsEnum.Runecraft) < pouch.level) continue;
-			if (bank.has(pouch.id)) inventorySize += pouch.capacity - 1;
-			if (bank.has(pouch.id) && pouch.id === itemID('Colossal pouch')) break;
-		}
+			// For each pouch the user has, increase their inventory size.
+			for (const pouch of Runecraft.pouches) {
+				if (user.skillLevel(SkillsEnum.Runecraft) < pouch.level) continue;
+				if (bank.has(pouch.id)) inventorySize += pouch.capacity - 1;
+				if (bank.has(pouch.id) && pouch.id === itemID('Colossal pouch')) break;
+			}
 
-		if (inventorySize > 28) boosts.push(`+${inventorySize - 28} inv spaces from pouches`);
+			if (inventorySize > 28) boosts.push(`+${inventorySize - 28} inv spaces from pouches`);
 
-		if (
-			user.skillLevel(SkillsEnum.Runecraft) >= 99 &&
-			user.hasEquippedOrInBank('Runecraft cape') &&
-			inventorySize > 28
-		) {
-			tripLength *= 0.97;
-			boosts.push('3% for Runecraft cape');
+			if (
+				user.skillLevel(SkillsEnum.Runecraft) >= 99 &&
+				user.hasEquippedOrInBank('Runecraft cape') &&
+				inventorySize > 28
+			) {
+				tripLength *= 0.97;
+				boosts.push('3% for Runecraft cape');
+			}
+		} else if (runners) {
```

Comment:
Doesnt require Else if (runners)

else would suffice

Reply:
does that work even if runners can be undefined?

Reply:
resolved